src/components/RankCategoryContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import { Center, Flex } from '@chakra-ui/react'
+import YoutubeApiWrapper from '../apis/youtubeApiWrapper'
+import Panel from './Panel'
+
+type Props = {
+    categoryName: string
+}
+
+const MAX_RESULTS = 1 //表示件数
+
+const RankCategoryContainer = ({ categoryName }: Props) => {
+    const [apiItems, setApiItems] = useState('')
+    const [videoApiItems, setVideoApiItems] = useState('')
+    const indexArray = [...Array(MAX_RESULTS)].map((_, index) => index)
```

Comment:
サルでもわかる変数名にしてほしい気持ちでいっぱいです。

```suggestion
    const indexNumberArray = [...Array(MAX_RESULTS)].map((_, index) => index)
```